src/main.rs

```diff
@@ -215,6 +223,11 @@ fn main() {
             .help("Proportion of removable points to remove (0-1 or 0%-100%)")
             .takes_value(true)
             .default_value(&"0.01"))
+        .arg(Arg::with_name("precision")
```

Comment:
It seems like this one can change the area of a polyline significantly, so not sure we'd want to have a default. It would probably be best to use clap's [`required_if`](https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.required_if) here, especially since we wouldn't be able to rely on a file extension default. It could be something like this

```rust
.required_if("format", "polyline")
```